clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -49,9 +49,11 @@
 
     public static final long TOPIC_EXPIRY_MS = 5 * 60 * 1000;
     private static final long TOPIC_EXPIRY_NEEDS_UPDATE = -1L;
+    private static final long DEFAULT_RETRY_BACKOFF_MS = 100L;
+    private static final long DEFAULT_METADATA_MAX_AGE_MS = 300 * 1000L;
```

Comment:
Would it be better to put that in ConsumerConfig so that we can reference the constant when defining METADATA_MAX_AGE_CONFIG in ConsumerConfig?